readme.md

```diff
@@ -154,6 +154,22 @@ alfy.cache.get('unicorn');
 //=> '🦄'
 ```
 
+##### maxAge
+
+The `set` method of this instance accepts an optional third argument where you can provide a `maxAge` option. `maxAge` is
+the number of milliseconds the value is valid in the cache.
+
+Example:
+
+```js
+alfy.cache.set('foo', 'bar', {maxAge: 5000});
+
+await delay(5000);
```

Comment:
Users wouldn't know what `delay` is. I would do `await require('delay')(5000);` to be explicit where it came from.